src/tests/tests.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter } from 'react-router-dom';
+import DetailsPage from '../components/DetailsPage';
+import Header from '../components/Header';
+import AnimesList from '../components/AnimesList';
+import store from '../redux/configureStore';
+import Anime from '../components/Anime';
+
+const MockObj = {
+  name: 'test',
+  image: 'imange',
+  id: 123,
+};
+
+describe('Detailed Anime component', () => {
```

Comment:
- [ ] Please, kindly add more tests to your app, as you should do some unit and integration tests as well.

> App has unit tests for several components of the app and some pure functions in Redux code.
> App has integration tests using React Testing Library
>   * Student can perform a mocking data fetching and Redux Store connection.

Reply:
done too